x-pack/solutions/search/plugins/search_indices/public/plugin.ts

```diff
@@ -93,6 +94,7 @@ export class SearchIndicesPlugin
 
     return {
       enabled: true,
+      fetchIndicesStatus: useIndicesStatusQuery.bind(null, queryClient),
```

Comment:
we should try to not do this IMO. This code is added to the page load bundle with will now include all the hook and react query code to support this.

I would just implement a fetch to this endpoint in `enterprise_search` instead.